integration/eclipse/src/i18n.ts

```diff
@@ -0,0 +1,22 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import { enTranslation, enCommonTranslation, deTranslation, deCommonTranslation } from '@axonivy/process-editor';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+export const initTranslation = () => {
+  if (i18n.isInitializing || i18n.isInitialized) return;
+  i18n
+    .use(initReactI18next)
+    .use(LanguageDetector)
+    .init({
+      debug: true,
+      supportedLngs: ['en', 'de'],
+      fallbackLng: 'en',
+      ns: ['process-editor'],
+      defaultNS: 'process-editor',
+      resources: {
+        en: { 'process-editor': enTranslation, common: enCommonTranslation },
+        de: { 'process-editor': deTranslation, common: deCommonTranslation }
+      }
+    });
+};
```

Comment:
seem to be duplicated in each integration -> make generic one?

Reply:
yes the "problem" is that e.g. the languagedetector is only needed in the integration, and we do not have a common integration package currently. but maybe this would make sense...

Reply:
I will proceed, and maybe do this later...